rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -39,6 +39,7 @@
 
 
 DEFAULT_TIMEOUT = 300
+SAWTOOTH_VERSION = '0.8.0'
```

Comment:
This should reference the VERSION file in the repo instead: https://github.com/hyperledger/sawtooth-core/blob/master/VERSION